package.json

```diff
@@ -41,11 +41,11 @@
     "e2e": "playwright test --trace on",
     "e2e:ui": "playwright test --ui",
     "e2e:update": "npm install -D @playwright/test@latest; npx playwright install --with-deps; npx playwright --version",
-    "create-env": "DB_PASS=$(openssl rand -base64 12) DATA_IMAGE_TAG=$(npm run docker:data-image-tag -s) envsubst < .env.example > .env",
+    "create-env": "[ ! -f .env ] && DB_PASS=$(openssl rand -base64 12) DATA_IMAGE_TAG=$(npm run docker:data-image-tag -s) envsubst < .env.example > .env",
```

Comment:
Only create the `.env` file if one does not already exist. This allows the dev to configure the connection to the containerized database via a GUI tool once, rather than requiring them to update the creds each time `create-env` is run.